examples/catalyst_contrastive/barlow_twins.py

```diff
@@ -0,0 +1,169 @@
+# flake8: noqa
+import argparse
+from functools import partial
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import top_k_accuracy_score
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data import DataLoader
+import torchvision
+from torchvision import transforms
+from torchvision.models.resnet import resnet50
```

Comment:
could you please check unused imports? it will be much easier to understand

Reply:
Sure